From 262a34ad11f7c0f6ed4e13cd45d37e88ac946525 Mon Sep 17 00:00:00 2001
From: Ohad Ben-Cohen <ohad@wizery.com>
Date: Tue, 13 Dec 2011 12:17:59 +0200
Subject: [PATCH] remoteproc: remove unused resource type

RSC_VIRTIO_CFG isn't being used, so remove it.

Originally it was introduced to overcome a resource table limitation
that prevented describing a virtio device in a single resource table
entry.

The plan though is to describe resource table entries in a TLV fashion,
where each entry will consume the amount of space it requires,
so the original limitation is anyway temporary.

Reported-by: Stephen Boyd <sboyd@codeaurora.org>
Signed-off-by: Ohad Ben-Cohen <ohad@wizery.com>
---
 include/linux/remoteproc.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/include/linux/remoteproc.h b/include/linux/remoteproc.h
index 1edbfde..b52f784 100644
--- a/include/linux/remoteproc.h
+++ b/include/linux/remoteproc.h
@@ -122,7 +122,6 @@ enum fw_resource_type {
 	RSC_TRACE	= 2,
 	RSC_VRING	= 3,
 	RSC_VIRTIO_DEV	= 4,
-	RSC_VIRTIO_CFG	= 5,
 };
 
 /**
-- 
1.7.0.4

